src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTask.java

```diff
@@ -174,14 +174,24 @@ private void processZipFileIfEligible(CloudBlobContainer container, String zipFi
         Envelope existingEnvelope =
             envelopeProcessor.getEnvelopeByFileAndContainer(container.getName(), zipFilename);
 
-        if (existingEnvelope != null) {
+        if (existingEnvelope != null && !existingEnvelope.getStatus().equals(Status.CREATED)) {
             log.warn(
-                "Envelope for zip file {} (container {}) already exists. Aborting its processing. Envelope ID: {}",
+                "Envelope for zip file {} (container {}) with {} status already exists. "
+                    + "Aborting its processing. Envelope ID: {}",
```

Comment:
Perhaps it would be more suitable to add envelope status in the last part? (after ID)